cmake/projects/Boost/schemes/url_sha1_boost_ios_library.cmake.in

```diff
@@ -345,7 +348,7 @@ ExternalProject_Add(
     INSTALL_COMMAND
     "${CMAKE_COMMAND}" -E make_directory "@HUNTER_PACKAGE_INSTALL_PREFIX@/lib"
     COMMAND
-    "@HUNTER_SELF@/scripts/create-boost-ios-fatlib.sh"
+    "@HUNTER_GLOBAL_SCRIPT_DIR@/create-boost-ios-fatlib.sh"
```

Comment:
Should be `HUNTER_PACKAGE_SCRIPT_DIR` instead of `HUNTER_GLOBAL_SCRIPT_DIR`.

Test failed:
* https://s3.amazonaws.com/archive.travis-ci.org/jobs/114699961/log.txt